allofplos/plos_corpus.py

```diff
@@ -569,15 +572,15 @@ def remote_proofs_direct_check(tempdir=newarticledir, article_list=None, plos_ne
     return proofs_download_list
 
 
-def download_check_and_move(article_list, text_list, tempdir, destination,
+def download_check_and_move(article_list, list_path, tempdir, destination,
```

Comment:
could `list_path` instead be `proof_filepath`? i'm just realising that list_path is way too generic to be clear in the signature of this function

Reply:
wait… why does list_path need to be here at all? it doesn't look like its used at all